pages/en/node-operators/transaction-failures.mdx

```diff
@@ -0,0 +1,53 @@
+import Page from "@reason/pages/Docs";
+import DocLink from "@reason/components/DocLink";
+export default Page ({ title: "Transaction failures" });
+
+# Transaction failures
+
+If a transaction fails to be applied, usually it would trigger 1 or more failures. There's 3 kind of transactions: payments, delegations, or zkApp transactions. There're several different kinds of failure in mina system. Some of the failures are specific to some kind of transactions. Let me explain them one by one below.
```

Comment:
suggested wording changes:
- "... it will trigger failures reported in the transaction response"
- "there are three kinds of ..."
- "there are several kinds ..."
- "We explain each one below"